src/pages/ReportSettingsPage.js

```diff
@@ -170,7 +156,7 @@ class ReportSettingsPage extends Component {
                             <OfflineWithFeedback
                                 pendingAction={lodashGet(this.props.report, 'pendingFields.reportName', null)}
                                 errors={lodashGet(this.props.report, 'errorFields.reportName', null)}
-                                onClose={this.resetToPreviousName}
+                                onClose={() => Report.clearPolicyRoomNameErrors(this.props.report.reportID)}
```

Comment:
Previously if we close the error indicator will clear the error and revert back to the previous room name but our Form component doesn't have the feature for that. What do you think about this? @puneetlath @jasperhuangg 

https://user-images.githubusercontent.com/25520267/210254224-6e0ad3d8-ba9c-4c8a-9a2e-3ade8b6169c3.mov

Reply:
To test this I comment the `ValidationUtils.isExistingRoomName` validation and update the room name with the used room name.

Reply:
@mollfpr some point to mention :

1. Closing the report settings screen will resets the value to the previous room name
2. The validation will prevent from submission as the room name already used (why you commented this `ValidationUtils.isExistingRoomName` ? )
3. This function is used with offline mode , check the second online test

Reply:
@mollfpr this is a good point , the actual form doesn't allow resetting values . this sounds like a good opportunity to implement this feature. Let's hear feedbacks from @puneetlath and @jasperhuangg

Reply:
I think it's fine that you can't reset the value. However, I don't think it should throw an error if you try to save it with the same name as the current name.

Reply:
>I don't think it should throw an error if you try to save it with the same name as the current name.

It doesn't throw an error , it will do nothing is this case

Reply:
@puneetlath here we prevent validation 
https://github.com/Expensify/App/blob/affbd27c9c1533b58f7506656a6c98119355d4f8/src/pages/ReportSettingsPage.js#L88-L91
and here we prevent form submission 
https://github.com/Expensify/App/blob/affbd27c9c1533b58f7506656a6c98119355d4f8/src/pages/ReportSettingsPage.js#L74-L77

Reply:
Ok cool. I think it's fine then that we don't have the behavior of canceling the error and re-setting the field. Thanks for checking @mollfpr.

Reply:
>13. Click close button of that error and verify that the error dismissed

Actually we have the behavior of canceling the error (dismissing) and it's added on offline tests as well , we just don't have re-setting the field

Reply:
I agree, I think it's fine we don't need to reset the field. I tried it myself and the behavior seems fine to me. @mollfpr Can you please update the screen recordings to include your offline tests as well? Please make sure you record both the offline and online clients so it's clear what's happening.